public/env-config.js

```diff
@@ -0,0 +1,8 @@
+window._env_ = {
+  REACT_APP_FILESIZE: '268435456',
+  REACT_APP_API_URL: 'https://dft-api.int.demo.catena-x.net/api',
+  REACT_APP_DEFAULT_COMPANY_BPN: 'BPNL0000000007UH',
+  REACT_APP_CLIENT_ID: 'Cl15-CX-DFT',
+  REACT_APP_KEYCLOAK_URL: 'https://centralidp.int.demo.catena-x.net/auth',
```

Comment:
real data should not be a default but the filename env-config.js implicates to me that this is your default for catena-x. I would suggest to create a sane default non working and a second one for catenax and name it